### ISSUE TITLE
Pyze is not working anymore (16-MAR-22?)

### ISSUE THREAD

**Issue by staudtm** [jamesremuscat/pyze#109]
I heard that Renault has changed the API (or introduced a new API key?). Can someone please check?

**Comment by staudtm** [comment]
I found this hint in a german forum (OpenWB):

New api-key.

$kamereon_api = 'VAX7XYKGfa92yMvXculCkEFyfZbuM7Ss';

**Comment by RalfJahns** [comment]
Thank you very much to staudtm! Was a quick fix because of this post.

**Comment by staudtm** [comment]
Hi Ralf, can you tell us how you fixed it in detail? Do we have to change the source code?

**Comment by RalfJahns** [comment]
No, no source code change required. You just have to use the new key.

From "CLI quickstart" section:

export KAMEREON_API_KEY=oF09WnKqvBDcr...
export GIGYA_API_KEY=3_e8d4g4SE_Fo8ahyH...

pyze login  # You should only need to do this once

You just have to do the above lines (again).

**Comment by TheReiner** [comment]
Same problem for me.. Until 16-MAR-22
But this solution doesn't work for me :-(

**Comment by TheReiner** [comment]
> 

Work fine with this Key ;)

**Comment by TheReiner** [comment]
> I found this hint in a german forum (OpenWB):
> 
> New api-key.
> 
> $kamereon_api = 'VAX7XYKGfa92yMvXculCkEFyfZbuM7Ss';

Thanks, Works fine with this new key :)

**Comment by berti2911** [comment]
Thanks to "TheReiner" for presenting this new kamereon key. I searched again at least 3 hours for solving the authorisation  problem. The new key works but how long.. this is a real shit.

**Comment by saxos1983** [comment]
Thank you very much TheReiner for finding this out.
